src/oxygen/robot_interface.py

```diff
@@ -257,7 +257,8 @@ def spawn_robot_keyword(self,
 
         for message in messages:
             if message:
-                robot_keyword.messages.append(RobotResultMessage(message))
+                ishtml = message[:6] == '*HTML*'
```

Comment:
instead of this, maybe use https://docs.python.org/3/library/stdtypes.html#str.startswith